keyboards/ning/tiny_board/tb16_rgb/info.json

```diff
@@ -0,0 +1,48 @@
+{
+    "manufacturer": "Ning",
+    "keyboard_name": "TB16 RGB",
+    "maintainer": "ningjx",
+    "bootloader": "caterina",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
```

Comment:
```suggestion
        "nkro": true,
        "rgb_matrix": true
```

Reply:
I have fixed all your suggestions,can you help me to review my code again,thanks